solr/solrj/src/java/org/apache/solr/common/util/ExecutorUtil.java

```diff
@@ -196,7 +198,13 @@ public void execute(final Runnable command) {
 
       String ctxStr = contextString.toString().replace("/", "//");
       final String submitterContextStr = ctxStr.length() <= MAX_THREAD_NAME_LEN ? ctxStr : ctxStr.substring(0, MAX_THREAD_NAME_LEN);
-      final Exception submitterStackTrace = enableSubmitterStackTrace ? new Exception("Submitter stack trace") : null;
+      final Exception submitterStackTrace;
```

Comment:
Why create Exception specifically -- why not do Throwable?

Reply:
No specific reason, I assume Exception is just the default for a lot of people and less cognitive burden when reading it.